src/copilot/utils/dataPlaneUtil.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import * as vscode from 'vscode';
+import { ApiCenterDataPlaneService, DataPlaneAccount } from "../../azure/ApiCenter/ApiCenterDataPlaneAPIs";
+import { AzureDataSessionProviderHelper, generateScopes } from "../../azure/azureLogin/dataSessionProvider";
+import { DataPlaneAccountsKey } from "../../constants";
+import { ext } from "../../extensionVariables";
+import { getSubscriptionContext } from "../../tree/DataPlaneAccount";
+import { UiStrings } from '../../uiStrings';
+import { GeneralUtils } from "../../utils/generalUtils";
+
+export async function createApiCenterDataPlaneService(): Promise<ApiCenterDataPlaneService> {
+    const account = await getOrSelectActiveAccount();
+    const scopes = generateScopes(account.clientId, account.tenantId);
+    const authSession = await AzureDataSessionProviderHelper.getSessionProvider().getAuthSession(scopes);
+
+    if (GeneralUtils.failed(authSession)) {
```

Comment:
actually, here can invoke 
```
const scopes = generateScopes(clientId, tenantId);
await AzureDataSessionProviderHelper.getSessionProvider().signIn(scopes);
``` 
to let user sign in this data plane. you may consider to let user sign in